etcdserver/api/v3rpc/grpc.go

```diff
@@ -36,7 +36,10 @@ func Server(s *etcdserver.EtcdServer, tls *tls.Config) *grpc.Server {
 	}
 	opts = append(opts, grpc.UnaryInterceptor(newUnaryInterceptor(s)))
 	opts = append(opts, grpc.StreamInterceptor(newStreamInterceptor(s)))
-
+	// grpc defaults to 4MB for received message length.
+	if s.Cfg.MaxRequestBytes > 4*1024*1014 {
```

Comment:
By default grpc set it at 4MB, I was thinking about if the MaxRequestBytes is > than 4MB, then increase it or else leave it as it is.